packages/inngest/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@skillrecordings/inngest",
```

Comment:
so we can use inngest across packages/apps it is a package

functions could also exist locally in apps, or even exported from other packages.